### ISSUE TITLE
Bugfixes for release 8

### ISSUE THREAD

**Issue by BurkovBA** [RNAcentral/rnacentral-webcode#251]
- [x] Clicking on Ensembl Transcripts in Genome browser does not work: http://test.rnacentral.org/genome-browser?species=homo_sapiens&chromosome=X&start=73819307&end=73856333 - (on the sequence page in the popup header there's a link to E!, which has some variable is undefined)
- [x] Fix pagination in publications:
http://localhost:8000/rna/URS000075D95B/9606
- [x] Slick gallery issue: due to double-loaded jquery?
- [x] Slick gallery styles
- [x] hsa-mir-548i-1 precursor leaks angular template
- [x] transcript link is not there for Gencode: ENST00000626826.1 (http://localhost:8000/rna/URS00002FE5F2/9606)
- [x] http://localhost:8000/rna/URS000075B4ED/9606 - I did find a problem in this page now - the RefSeq entry lists `alternative transcript`
- [x] database logos (check capital and small-size letters in database labels)
- [x] Fonts on the badges of database tooltips
- [x] Stars on the badges of databases disappeared
- [x] navbar 50px
- [x] dropdown menu elements colors
- [x] import leftover models into `__init__.py`
- [x] taxonomic tree on sequence page won't scroll horizontally!
- [x] Silva nucleotides are repeated twice in the template: http://localhost:8000/rna/URS00005B2216/511145
- [x] Show each distinct `ChemicalComponent` in modifications only once or the list gets huge, e.g. http://localhost:8000/rna/URS000043B91E/9606
- [x] View genomic location does not work on this page: http://test.rnacentral.org/rna/URS000075A823/9606
- [x] Page with deleted Xrefs only: need to filter entries by taxid, show warnings and color Xrefs with yellow, e.g.: http://localhost:8000/rna/URS00002EF971/10090
- [x] Fix forna preventDefault on taxonomy tree species links (they became unclickable)
- [x] Scroll to top, when in Xrefs user clicks on 'All publications'
- [x] Remove "show all" with magnifying glass in AbstractComponent on the database page
- [x] restore Greengenes custom taxonomic lineage that used to be shown in the Annotations table

WON'T DOs:

- [x] sequence only lists HGNC but there are 3 more active xrefs (`select * from xref where upi = 'URS0000572B72' and deleted = 'N';`) http://localhost:8000/rna/URS0000572B72/9606 - WON'T FIX, we're intentionally filtering-out ENA by project_id
- [x] similarly: The counts of databases a sequence is in seems unreliable. Consider http://localhost:8000/rna/URS000030C9DE/9606 which says it is present in 3 databases but I only see 2. I've seen examples where the count is 0 - WON'T FIX, we're intentionally filtering-out ENA by project_id
- [x] Filter publications by taxid - moved to #199 
- [x] Genome browser wouldn't load in Safari - already in #230 
- [x] Anchors on api documentation pages are redirecting to the homepage - fixed in #171
- [x] flash of unstyled content - poked around, will be solved by #230
- [x] 2 managers on Xref - moved to #253 
- [x] broken jsonp in EuroPMC on URS000075C808/9606 - moved to #254

**Comment by AntonPetrov** [comment]
- [ ] Zoom out Genoverse on sequence pages

**Comment by AntonPetrov** [comment]
- [ ] Silva nucleotides are listed twice:
http://localhost:8000/rna/URS00005B2216/511145

**Comment by BurkovBA** [comment]
`Zoom out Genoverse on sequence pages`

Yep, zoom out is a TODO in #110.

**Comment by AntonPetrov** [comment]
- [ ] _View genomic location_ does not work on this page:
http://test.rnacentral.org/rna/URS000075A823/9606

**Comment by blakesweeney** [comment]
- [ ] Silva links should use https. They redirect to https from http but would prefer https.

**Comment by AntonPetrov** [comment]
- [ ] restore Greengenes custom taxonomic lineage that used to be shown in the Annotations table
